<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WinLeft</class>
 <widget class="QWidget" name="WinLeft">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>417</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#WinLeft{
	border-image: url(:/image/background/mainwin.png);
}
/*-------------------------滑动条部分--------------------*/
QSlider::groove:horizontal{
	border: 1px;
	height: 3px; 
	position: absolute;
	border-radius :1px;
	margin: 1px 0;
	left: 2px; right: 2px;
	background-color: rgb(164, 164, 164);
}
/*滑动手柄部分*/
QSlider::handle:horizontal:enabled{
	background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #a4c4f8, stop:1 #68a3ed);
    border: 1px;
    width: 15px;
    height: 10px;
    margin: -4px -1;
    border-radius: 5px;
}
QSlider::handle:horizontal:!enabled{
background: qlineargradient(x1:0, y1:0, x2:0, y2:1, stop:0 #cccccc, stop:1 #bfbfbf);
    border: 1px;
    width: 15px;
    height: 10px;
    margin: -4px -1;
    border-radius: 5px;
}
QLabel{
	color: rgb(226, 222, 222);
}


/*按钮部分*/
QPushButton:enabled{
	background-color: rgb(67, 67, 67);
	color: rgb(226, 222, 222);
	border:1px solid #5a5a5a;
}
QPushButton:!enabled{
	background-color: rgb(88, 88, 88);
	color: rgb(226, 222, 222);
	border:1px solid #5a5a5a;
}
QPushButton:checked{
	color: rgb(236, 236, 236);
background-color: rgb(54, 54, 54);

}
QPushButton::hover{
	background-color: rgb(54, 54, 54);
}
QLine{
	background-color: rgb(154, 154, 154);
}</string>
  </property>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>0</y>
     <width>1</width>
     <height>417</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(154, 154, 154);
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>100</y>
     <width>66</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Hue</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>220</y>
     <width>66</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Gamma</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_Gamma">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>220</y>
     <width>92</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_Hue">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>100</y>
     <width>92</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_WB">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>300</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>WhiteBalance</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_Contrast">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>92</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>60</y>
     <width>66</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Contrast</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_Brightness">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>92</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>20</y>
     <width>66</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Brightness</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_Saturation">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>140</y>
     <width>92</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_18">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>140</y>
     <width>66</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Saturation</string>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_Sharpness">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>180</y>
     <width>92</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_19">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>180</y>
     <width>66</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Sharpness</string>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>380</y>
     <width>200</width>
     <height>1</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(154, 154, 154);
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_25">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>388</y>
     <width>66</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Flicker</string>
   </property>
  </widget>
  <widget class="QComboBox" name="ComboBox_Flicker">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>391</y>
     <width>55</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::WheelFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox{
	border:0px;
	border-radius:4px;
	color: rgb(112, 112, 112);
}

QComboBox::drop-down {
      subcontrol-origin: padding;
      subcontrol-position: top right;
      width: 20px;
      border-left-width: 0px;
      border-left-color: darkgray;
      border-left-style: solid; /* just a single line */
      border-top-right-radius: 4px; /* same radius as the QComboBox */
      border-bottom-right-radius: 4px;
  }
QComboBox::down-arrow {
    
	border-image: url(:/image/icon/arrow.png);
}</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <item>
    <property name="text">
     <string>OFF</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>50Hz</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>60Hz</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="BT_Mirror">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>140</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Mirror</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_Flip">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>100</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Flip</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_HOME">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>220</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Home</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_Menu">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>180</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>7</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::CustomContextMenu</enum>
   </property>
   <property name="text">
    <string>Menu OFF</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_Reset">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>340</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_11">
   <property name="geometry">
    <rect>
     <x>267</x>
     <y>0</y>
     <width>1</width>
     <height>417</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">	background-color: rgb(154, 154, 154);
</string>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_PUSH">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>300</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Push</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_AI">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>260</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Framing</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="v_brightness">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>20</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border-radius:3px;qproperty-alignment:AlignHCenter;</string>
   </property>
   <property name="text">
    <string>10000</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="v_contrast">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>60</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border-radius:3px;qproperty-alignment:AlignHCenter;</string>
   </property>
   <property name="text">
    <string>10000</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="v_hue">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>100</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border-radius:3px;qproperty-alignment:AlignHCenter;</string>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="v_saturation">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>140</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border-radius:3px;qproperty-alignment:AlignHCenter;</string>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="v_sharpness">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>180</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border-radius:3px;qproperty-alignment:AlignHCenter;</string>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="v_gamma">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>220</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border-radius:3px;qproperty-alignment:AlignHCenter;</string>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="v_wb">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>340</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border-radius:3px;qproperty-alignment:AlignHCenter;</string>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QSlider" name="Slider_BLC">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>260</y>
     <width>92</width>
     <height>16</height>
    </rect>
   </property>
   <property name="maximum">
    <number>20</number>
   </property>
   <property name="pageStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLineEdit" name="v_blc">
   <property name="geometry">
    <rect>
     <x>165</x>
     <y>260</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background:transparent;
color: rgb(255, 255, 255);
border-radius:3px;qproperty-alignment:AlignHCenter;</string>
   </property>
   <property name="text">
    <string>1000</string>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>260</y>
     <width>66</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>BackLight</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <bold>false</bold>
     <stylestrategy>PreferDefault</stylestrategy>
    </font>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="Cb_WB">
   <property name="geometry">
    <rect>
     <x>178</x>
     <y>303</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_IR">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>20</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>IR OFF</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_DAY">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>60</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>DAY</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="BT_RST_PTZ">
   <property name="geometry">
    <rect>
     <x>215</x>
     <y>380</y>
     <width>48</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>8</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>RST-PTZ</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
